### PR TITLE
[IMP] mexico.rst: Completed documentation about the payment term

### PR DESCRIPTION
**Pull request by luistorresm**
Based on this change:

https://github.com/odoo/enterprise/pull/2993

### CODE REVIEW

File: accounting/localizations/mexico.rst [hunk 1]
```diff
@@ -262,12 +262,40 @@ If the invoice is not paid.
...
+   - If today is the first day of the month and I generate an invoice with the
+     `Payment Term` `30 Net Days` I could see that the `Due Date` is less than
+     the day 17 of the next month, then the payment term is `PPD`.
+
+   - If today I generate an invoice that the `Payment Term` `30 Net Days` and
+     the `Due Date` is higher than the day 17 of the next month the payment
+     term is `PPD`.
+
+   - If I has a `Payment Term` with 2 lines or more, example
+     `30% Advance End of Following Month`, this is term in partialities, then
```
Comment (luisg123v) [inline accounting/localizations/mexico.rst]: that is term in partialities, thus

File: accounting/localizations/mexico.rst [hunk 2]
```diff
@@ -262,12 +262,40 @@ If the invoice is not paid.
...
+   **1.1. How can I generate an invoice with payment term `PUE`?**
+
+   `According to the SAT documentation`_ if the payment was agreed that the
+   amount total that covers the document will be received before that the
+   day 17 of the month calendar immediately after of the month that was
+   issued the CFDI.
+
+   **1.2. How can I get this with Odoo?**
+
+   In Odoo is very easy to indicate when an invoice must be PPD or PUE, is
```
Comment (GavyMG) [inline accounting/localizations/mexico.rst]: ..PPD or PUE, **it's**..

File: accounting/localizations/mexico.rst [hunk 3]
```diff
@@ -262,12 +262,40 @@ If the invoice is not paid.
...
+   based on the `Payment Terms` defined in the invoice.
+
+   - If I do not define a `Payment Term` in the invoice the payment term must
+     be `PPD`.
```
Comment (GavyMG) [inline accounting/localizations/mexico.rst]: be `PUE`.

File: accounting/localizations/mexico.rst [hunk 4]
```diff
@@ -262,12 +262,40 @@ If the invoice is not paid.
...
+
+   - If today is the first day of the month and I generate an invoice with the
+     `Payment Term` `30 Net Days` I could see that the `Due Date` is less than
+     the day 17 of the next month, then the payment term is `PPD`.
```
Comment (GavyMG) [inline accounting/localizations/mexico.rst]: ...then the payment term is `PUE`.

Maybe add that the payment term is for one payment (no partialities)

File: accounting/localizations/mexico.rst [hunk 5]
```diff
@@ -262,12 +262,40 @@ If the invoice is not paid.
...
+   `According to the SAT documentation`_ if the payment was agreed that the
+   amount total that covers the document will be received before that the
+   day 17 of the month calendar immediately after of the month that was
+   issued the CFDI.
+
+   **1.2. How can I get this with Odoo?**
+
+   In Odoo is very easy to indicate when an invoice must be PPD or PUE, is
+   based on the `Payment Terms` defined in the invoice.
+
```
[the remark this answers is not in the document]
Comment (hbto) [reply accounting/localizations/mexico.rst]: This sentence needs better wording.

In order to set the `Here you set appropriate concept` in invoice with PPD or PUE, you can easily set it by using the `Payment Terms` defined in the invoice.

File: accounting/localizations/mexico.rst (hunk 4 above)
Comment (hbto) [reply accounting/localizations/mexico.rst]: Follow this example #310 (comment)

File: accounting/localizations/mexico.rst (hunk 3 above)
Comment (hbto) [reply accounting/localizations/mexico.rst]: Follow this example #310 (comment)

**Comment by luistorresm** [comment]
Done @hbto @luisg123v @GavyMG 

Thanks

**Comment by luistorresm** [comment]
@moylop260 @nhomar 

Could you approve this, please?

A cat is dying by this information.

Regards

**Pull request merged**